module-bsp/sink/FileSink.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "sink/FileSink.hpp"
+
+FileSink::FileSink(const std::string &filepath)
+{
+    fs.open(filepath, std::fstream::out);
+}
+
+FileSink::~FileSink()
+{
+    fs.close();
+}
+
+void FileSink::put(const char *data, size_t size)
+{
+    fs.write(data, size);
+}
```

Comment:
it's it's not used at the moment in code - as sink is selected in build time ( not in runtime via configuration :) )
I think it should be changed and possibly expanded so that we would have config which would set overall PowerManagement setup: select sinks for selected printer, set power profiles for cpu calling etc.